cfgov/permissions_viewer/tests/test_views.py

```diff
@@ -0,0 +1,61 @@
+from django.test import TestCase, Client
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group, Permission
+from django.core.urlresolvers import reverse
+
+User = get_user_model()
+
+
+class TestPermissionsViews(TestCase):
+    def setUp(self):
+        self.testuser = User.objects.create_superuser(
+            username='foo',
+            password='bar',
+            email='foo@cfpb.gov')
+        self.testuser.save()
+
+        self.testgroup = Group.objects.create(name='Developers')
+        self.testgroup.user_set.add(self.testuser)
+
+    def test_user_has_page(self):
+        client = Client()
```

Comment:
This line (and others like it) are unnecessary if you use `django.TestCase` as you do here, as [that class includes an instance of `Client`](https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.SimpleTestCase.client). You should be able to use `self.client`.

Reply:
good call

Reply:
this and the isort thing are addressed in the latest commit